README.md

```diff
@@ -15,3 +16,60 @@
 
 ### Run unit tests
     yarn test:unit
+
+## Using this library
+
+Package requirements, to be added after `vue create ...`
+
+```bash
+# Vuetify
+vue add vuetify
+
+# Vee Validate
+yarn add vee-validate
```

Comment:
vee-validate is 84KB minified / 23KB gzipped, so we should make a compelling case for bringing it in. Vuetify already provides some nice utilities for validation, what features of vee-validate do we need beyond that? Is it something we could add inexpensively without pulling in a third party library?

ping @jbeezley since I think he also has experience with vee-validate.

Reply:
There are ways to [reduce the size](https://baianat.github.io/vee-validate/concepts/bundle-size.html#minimal-bundle).

I ended up using vee-validate for resonantgeo because I found the Vuetify's syntax to be excessively verbose, so I think it's something we could consider.  Here though, it's only being used to validate required components.  This can just be replaced by a prop (`:rules="[v => !!v || 'Item is required']"`) on the form element.

Another thing to note is that Vee-Validate [does not work](https://github.com/baianat/vee-validate/issues/1267) with vue-test-utils in synchronous mode.  For resonantgeo, I had to make all tests that touched the validation logic async.  It wasn't too bad, but it did require adding a lot of `await Vue.$nextTick()` statements.

For now, I would lean against using it until we have a more complicated use case.